Backend/DatabaseRequester.py

```diff
@@ -0,0 +1,58 @@
+import psycopg2
+from configparser import ConfigParser
+
+class DatabaseRequester:
+
+    conn = None
+    connected = False
+    host = None
+    port = None
+    database = None
+    user = None
+    password = None
+
+    # get database info from direct arguments
+    def __init__(self, host, port, database, user, password):
+        self.conn = None
+        self.connected = False
+        self.host = host
+        self.port = port
+        self.database = database
+        self.user = user
+        self.password = password
+
+    def connect(self):
+        if self.conn is None:
+            try:
+                self.conn = psycopg2.connect(
+                    host=self.host,
+                    port=self.port,
+                    database=self.database,
+                    user=self.user,
+                    password=self.password
+                )
+                self.connected = True
+                print("DB Connected")
+            except psycopg2.DatabaseError:
+                return
```

Comment:
would you mind please logging some kind of error message (or just printing it to console) here? otherwise the program will run into problems somewhere after this function and it won't be all that clear that this was the cause